templates/repo/diff/box.tmpl

```diff
@@ -165,6 +165,9 @@
 									{{if $file.IsDeleted}}
 										<a class="ui basic tiny button" rel="nofollow" href="{{$.BeforeSourcePath}}/{{PathEscapeSegments .Name}}">{{ctx.Locale.Tr "repo.diff.view_file"}}</a>
 									{{else}}
+										{{if and $.Repository.CanEnableEditor $.CanEditFile (not $file.IsLFSFile)}}
+											<a class="ui basic tiny button" rel="nofollow" href="{{$.HeadRepoLink}}/_edit/{{PathEscapeSegments $.HeadBranchName}}/{{PathEscapeSegments $file.Name}}?hide_create_pr=true&hide_create_new_branch=true&back_to_pr={{$.Issue.Index}}&back_to_diff_hash={{$file.NameHash}}">{{ctx.Locale.Tr "repo.editor.edit_this_file"}}</a>
+										{{end}}
```

Comment:
I think it's better to put the "Edit" button after the "View" button

Reply:
ec6ab8d2d5a8c2b01b7288fd1d7d58998e50de79